DataFormats/TotemRPReco/interface/TotemRPRecHit.h

```diff
@@ -0,0 +1,55 @@
+/****************************************************************************
+*
+* This is a part of TOTEM offline software.
+* Authors:
+*   Hubert Niewiadomski
+*   Jan Kašpar (jan.kaspar@gmail.com)
+*
+****************************************************************************/
+
+#ifndef DataFormats_TotemRPReco_TotemTotemRPRecHit
+#define DataFormats_TotemRPReco_TotemTotemRPRecHit
+
+/**
+ *\brief Reconstructed hit in TOTEM RP.
+ *
+ * Basically a cluster (TotemRPCluster), the position of which has been converted into actual geometry (in mm).
+ **/
+class TotemRPRecHit
+{
+ public:
+  TotemRPRecHit(unsigned int det_id, double position, double sigma) : det_id_(det_id), 
+    position_(position), sigma_(sigma) {}
+  TotemRPRecHit() : det_id_(0), position_(0), sigma_(0) {}
+
+  inline void Position(double position) {position_=position;}
```

Comment:
@slava77 We're planning fix this issue and also others related to CMS coding standards in few days. Can you provide first some comments on more general level, like code placement (i.e. are we using right packages to store the code) and design ?